packages/systemd/systemd-journald.conf

```
@@ -0,0 +1,2 @@
+[Service]
+Environment=SYSTEMD_JOURNAL_COMPACT=0
```

```suggestion
[Service]
Environment=SYSTEMD_JOURNAL_KEYED_HASH=0
Environment=SYSTEMD_JOURNAL_COMPACT=0
```

To drop patch 9007.

FWIW the reason I implemented it as a patch was because I wanted the patch to fail if the `SYSTEMD_JOURNAL_KEYED_HASH` option was ever removed upstream. That would be more obvious to a developer than an environment variable silently being ignored, though either regression should be caught by testing.

I'm fine with either approach but we should be consistent.